packages/search-in-workspace/src/browser/search-in-workspace-widget.tsx

```diff
@@ -316,19 +316,30 @@ export class SearchInWorkspaceWidget extends BaseWidget implements StatefulWidge
 
     protected renderReplaceFieldToggle(): React.ReactNode {
         const toggle = <span className={`fa fa-caret-${this.showReplaceField ? 'down' : 'right'}`}></span>;
+
+        const replaceToggle = () => {
+            const elArr = document.getElementsByClassName('replace-toggle');
+            if (elArr && elArr.length > 0) {
+                (elArr[0] as HTMLElement).focus();
+            }
+            this.showReplaceField = !this.showReplaceField;
+            this.resultTreeWidget.showReplaceButtons = this.showReplaceField;
+            this.update();
+        };
+
         return <div
             title='Toggle Replace'
             className='replace-toggle'
             tabIndex={0}
-            onClick={e => {
-                const elArr = document.getElementsByClassName('replace-toggle');
-                if (elArr && elArr.length > 0) {
-                    (elArr[0] as HTMLElement).focus();
+            onClick={() => {
+                replaceToggle();
+            }}
+            onKeyPress={e => {
+                if (e.key === 'Enter' || e.key === ' ') {
```

Comment:
What is `e.key === ' '`? What issue are you trying to solve here?

Reply:
Thanks! @kittaakos here I'm making sure that the users can use the `Space` key as well to do what they can do with `Enter` as  both `Enter` and `Space` keys can be used interchangeably to activate buttons.
 
Source: https://webaim.org/techniques/keyboard/

![image](https://user-images.githubusercontent.com/46004116/85947518-a8ea0a80-b964-11ea-826f-b65cfa5594f3.png)